pos-client-library/src/main/java/com/globalaccelerex/globalaccelerexandroidposclientlibrary/transactions/CardNotPresentTransactions.kt

```diff
@@ -31,6 +31,8 @@ class CardNotPresentTransactions(countryCode: Countries) {
         customPrint: Boolean,
         callingComponent: Any
     ) {
+        require(cardNumber.length in 12..18) {throw IllegalArgumentException("Card Number should be between 12 and 18 digits")}
```

Comment:
require already throws an IllegalArgumentException. No need to throw another one.

Reply:
This is to pass the information that caused the crash

Reply:
Yes, just write the message inside the lambda. Not need to throw any other exception. If you check the signature of require, it uses the string in the lambda to throw the exception

Reply:
I've updated it.